src/test/java/core/basesyntax/Train.java

```diff
@@ -3,6 +3,6 @@
 /**
  * Feel free to remove this class and create your own.
  */
-public class HelloWorld {
+public class Train {
```

Comment:
do we need this class here?

Reply:
It was my first idea and I forgot to delete it.